### PR TITLE
impl displayname check and display message

### PR DESCRIPTION
**Pull request by sureshcjc**

### CODE REVIEW

File: src/main/java/com/victorious/user/service/impl/UserServiceImpl.java [hunk 1]
```diff
@@ -44,9 +45,23 @@ public User getById(Long id) throws NotFoundException {
         }
         return users;
     }
+    
+    @Override
+    public List<User> searchUsersByDisplayName(String displayName) throws NotFoundException {
```
Comment (gtvictorious) [inline src/main/java/com/victorious/user/service/impl/UserServiceImpl.java]: why are we not letting the database handle this, by catching the fail to create or update? Seems unnecessary to have to query every user field to figure out if its valid or not

Comment (davidlaroue) [reply src/main/java/com/victorious/user/service/impl/UserServiceImpl.java]: Yeah good call.

Comment (vincentoe) [reply src/main/java/com/victorious/user/service/impl/UserServiceImpl.java]: Agreed, we should just use the database unique column enforcement and just bubble that up unless theres a clear reason not to.

Comment (sureshcjc) [reply src/main/java/com/victorious/user/service/impl/UserServiceImpl.java]: yes, that check exists, but the sql exception could fail on any column. we need to identify the field and generate specific display message.

Comment (gtvictorious) [reply src/main/java/com/victorious/user/service/impl/UserServiceImpl.java]: We will write custom validators into JPA later. For now this works.
We should bring it up in backend sync tho.

**Pull request merged**